src/collective/exportimport/import_content.py

```diff
@@ -495,6 +500,17 @@ def import_constrains(self, obj, item):
         immediately_addable_types = item["exportimport.constrains"]["immediately_addable_types"]
         constrains.setImmediatelyAddableTypes(immediately_addable_types)
 
+    def import_workflow_history(self, obj, item):
+        workflow_history = item.get("workflow_history", {})
+        result = {}
+        for key, value in workflow_history.items():
+            # The time needs to be deserialized
+            for history_item in value:
+                history_item["time"] = DateTime(dateutil.parser.parse(history_item["time"]))
```

Comment:
This will  correctly pick up time zone information as well? 

I recall having other issues with date parsing last yar in a custom import method to check and fix EffectiveDate being before CreationDate on an item.  

Maybe we should use the above as the canonical version and fix the ugly strptime hack that's still in here. -> 
https://github.com/collective/collective.exportimport/blob/2b73ac53f0b8387416ac39161e75a9843c79d4bb/src/collective/exportimport/import_content.py#L372-L381

Only other place we currently use dateutil in exportimport is for comments, should it as this place not also be casted to Zope's DateTime? https://github.com/collective/collective.exportimport/blob/efc71c583e53e22c77cf3dbb78844264286bbb24/src/collective/exportimport/import_other.py#L570

Reply:
In fact it does not. The timezone info is converted to UTC during serializing. See https://github.com/plone/plone.restapi/commit/d0df8512a720e4bf42468c6245210ab03d4d5ba5
For this usecase that seems fine. For fields like start/end the deserializer takes care of converting back to the right timezone though. We should switch the other uses of `datetime.strptime` to use dateutil.